{"uid":"d4288a00c4cbbbcf","name":"Validate Council Output -- @1.30 Testing : ValeofGlamorganCouncil","fullName":"Test each council output matches expected results in /outputs: Validate Council Output","historyId":"39580327bc77725bd95a447ff969d73d","time":{"start":1673741525262,"stop":1673741655627,"duration":130365},"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='myvale.valeofglamorgan.gov.uk', port=443): Max retries exceeded with url: /getdata.aspx?RequestType=LocalInfo&ms=ValeOfGlamorgan%2FAllMaps&group=Community+and+Living%7CRefuse+HIDE2&type=json&callback=AddressInfoCallback&uid=64029020&import=jQuery35108514154283927682_1673022974838&_=1673022974840 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3af1141820>: Failed to establish a new connection: [Errno 110] Connection timed out'))\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"uk_bin_collection/tests/features/steps/validate_council.py\", line 36, in step_impl\n    context.parse_result = collect_data.main(args)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 63, in main\n    return client_code(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 23, in client_code\n    return get_bin_data_class.template_method(address_url, **kwargs)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 55, in template_method\n    bin_data_dict = self.parse_data(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/councils/ValeofGlamorganCouncil.py\", line 49, in parse_data\n    response = requests.get('https://myvale.valeofglamorgan.gov.uk/getdata.aspx', params=params,\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 73, in get\n    return request(\"get\", url, params=params, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/adapters.py\", line 565, in send\n    raise ConnectionError(e, request=request)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='myvale.valeofglamorgan.gov.uk', port=443): Max retries exceeded with url: /getdata.aspx?RequestType=LocalInfo&ms=ValeOfGlamorgan%2FAllMaps&group=Community+and+Living%7CRefuse+HIDE2&type=json&callback=AddressInfoCallback&uid=64029020&import=jQuery35108514154283927682_1673022974838&_=1673022974840 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3af1141820>: Failed to establish a new connection: [Errno 110] Connection timed out'))\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"uk_bin_collection/tests/features/steps/validate_council.py\", line 36, in step_impl\n    context.parse_result = collect_data.main(args)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 63, in main\n    return client_code(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 23, in client_code\n    return get_bin_data_class.template_method(address_url, **kwargs)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 55, in template_method\n    bin_data_dict = self.parse_data(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/councils/ValeofGlamorganCouncil.py\", line 49, in parse_data\n    response = requests.get('https://myvale.valeofglamorgan.gov.uk/getdata.aspx', params=params,\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 73, in get\n    return request(\"get\", url, params=params, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/adapters.py\", line 565, in send\n    raise ConnectionError(e, request=request)\n","steps":[{"name":"Given the council: \"ValeofGlamorganCouncil\"","time":{"start":1673741525262,"stop":1673741525263,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"When we scrape the data from \"ValeofGlamorganCouncil\"","time":{"start":1673741525263,"stop":1673741655626,"duration":130363},"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='myvale.valeofglamorgan.gov.uk', port=443): Max retries exceeded with url: /getdata.aspx?RequestType=LocalInfo&ms=ValeOfGlamorgan%2FAllMaps&group=Community+and+Living%7CRefuse+HIDE2&type=json&callback=AddressInfoCallback&uid=64029020&import=jQuery35108514154283927682_1673022974838&_=1673022974840 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3af1141820>: Failed to establish a new connection: [Errno 110] Connection timed out'))\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"uk_bin_collection/tests/features/steps/validate_council.py\", line 36, in step_impl\n    context.parse_result = collect_data.main(args)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 63, in main\n    return client_code(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/collect_data.py\", line 23, in client_code\n    return get_bin_data_class.template_method(address_url, **kwargs)\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/get_bin_data.py\", line 55, in template_method\n    bin_data_dict = self.parse_data(\n  File \"/home/runner/work/UKBinCollectionData/UKBinCollectionData/uk_bin_collection/uk_bin_collection/councils/ValeofGlamorganCouncil.py\", line 49, in parse_data\n    response = requests.get('https://myvale.valeofglamorgan.gov.uk/getdata.aspx', params=params,\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 73, in get\n    return request(\"get\", url, params=params, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/uk-bin-collection-EwS6Gn8s-py3.9/lib/python3.9/site-packages/requests/adapters.py\", line 565, in send\n    raise ConnectionError(e, request=request)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},{"name":"Then the result is valid json","time":{"start":1673741655627,"stop":1673741655627,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"And the output should validate against the schema","time":{"start":1673741655627,"stop":1673741655627,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":4,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Test each council output matches expected results in /outputs"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"council","value":"ValeofGlamorganCouncil"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":1,"unknown":0,"total":3},"items":[{"uid":"4f0c249c981a302c","reportUrl":"https://robbrad.github.io/UKBinCollectionData/3.9/393//#testresult/4f0c249c981a302c","status":"broken","statusDetails":"requests.exceptions.ConnectionError: HTTPSConnectionPool(host='myvale.valeofglamorgan.gov.uk', port=443): Max retries exceeded with url: /getdata.aspx?RequestType=LocalInfo&ms=ValeOfGlamorgan%2FAllMaps&group=Community+and+Living%7CRefuse+HIDE2&type=json&callback=AddressInfoCallback&uid=64029020&import=jQuery35108514154283927682_1673022974838&_=1673022974840 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8f9e272cd0>: Failed to establish a new connection: [Errno 110] Connection timed out'))\n","time":{"start":1673740901812,"stop":1673741032248,"duration":130436}},{"uid":"7628716f4556b491","reportUrl":"https://robbrad.github.io/UKBinCollectionData/3.9/392//#testresult/7628716f4556b491","status":"passed","time":{"start":1673740456609,"stop":1673740460680,"duration":4071}}]},"tags":[]},"source":"d4288a00c4cbbbcf.json","parameterValues":["ValeofGlamorganCouncil"]}